cumulusci/core/tests/test_robotframework_utils.py

```diff
@@ -0,0 +1,33 @@
+import unittest
```

Comment:
Please move this file to `cumulusci/robotframework/tests/test_utils.py`. In CumulusCI we aim to put tests for Python modules in the tests directory of the folder where the module-under-test is.